docs/guides/customize-your-scene.mdx

```diff
@@ -0,0 +1,758 @@
+  ]);
+}
+```
+
+
+</details>
+
+#### `clearTransforms`
+
+- Parameters:
+  - `cascade?: boolean` - whether this operation should cascade to child items
+- Specifies that the items matching the query should have their overridden transformation matrix removed. The `cascade` flag determines whether
+  children of the items matching the query should also have their overridden transformation matrix removed, and defaults to `true`.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Clear the overridden the transformation matrix for the item with the `item-uuid` ID
+  // and do not cascade to preserve transformations on children
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).clearTransforms(false),
+  ]);
+
+  // Clear the overridden the transformation matrix for the item with the `item-uuid` ID
+  // and cascade to clear overridden transformations on children
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).clearTransforms(true),
+  ]);
+}
+```
+
+
+</details>
+
+#### `setPhantom`
+
+- Parameters:
+  - `phantomState?: boolean` - the phantom state to apply to the items
+- Specifies that the items matching the query should have their phantom state overridden to match the specified `phantomState` flag. If the
+  `phantomState` flag is not provided, it will default to `true`.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Mark the item with the `item-uuid` ID as phantom
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).setPhantom(true),
+  ]);
+
+  // Unmark the item with the `item-uuid` ID as phantom
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).setPhantom(false),
+  ]);
+}
+```
+
+
+</details>
+
+#### `clearPhantom`
+
+- Parameters:
+  - None
+- Specifies that the items matching the query should have their overridden phantom state removed.
+
+<details className="code">
+<summary>Example</summary>
+
+
+```js
+async function performOperation() {
+  const viewer = document.querySelector('vertex-viewer');
+  const scene = await viewer.scene();
+
+  // Clear the overridden phantom state of the item with the `item-uuid` ID
+  await scene.items((op) => [
+    op.where((q) => q.withItemId('item-uuid')).clearPhantom(),
+  ]);
+}
+```
+
+
+</details>
+
+#### `setEndItem`
+
+- Parameters:
+  - `endItemState?: boolean` - the end item state to apply to the items
+- Specifies that the items matching the query should have their end item state overridden to match the specified `endItemState` flag. If the
+  `endItemState` flag is not provided, it will default to `true`.
```

Comment:
💡 Should we consider adding a note here about how the end item changes do not propagate down the tree like other queries?

Reply:
👍 that's probably worth calling out. I'll update this!